### ISSUE TITLE
2.x bower not working

### ISSUE THREAD

**Issue by neffets** [heiseonline/shariff#260]
I cloned the git repo and wanted to use the fresh shariff js/css.
Becaues "build/" dir is missing I want to build it.

"grunt demo" - does not work

"bower install" shows only error messages:
```
bower                     invalid-meta for:/srv/www/imperia9/berlinonline/landmasterlayouts/vendor/heiseonline-shariff/bower.json
bower                     invalid-meta The "main" field cannot contain minified files
bower                     invalid-meta The "main" field cannot contain minified files
```

How to build the current CSS/js ? Please add hints to the README.md

**Comment by neffets** [comment]
I found "webpack" cmd line in package.json.

Is this the "new right way"?
```
sudo npm install -g webpack
rm -fr dist && webpack -p
```
then there are generated css/js files in the dist/ folder

**Comment by pmb0** [comment]
Fixed The problem was solved with v2.0.2.

The demo page can be started with `npm run dev`. The README has been adjusted accordingly. Webpack does *not* need to be installed globally.
